src/Microsoft.Net.Http.Headers/CacheControlHeaderValue.cs

```diff
@@ -11,18 +11,18 @@ namespace Microsoft.Net.Http.Headers
 {
     public class CacheControlHeaderValue
     {
-        private const string MaxAgeString = "max-age";
-        private const string MaxStaleString = "max-stale";
-        private const string MinFreshString = "min-fresh";
-        private const string MustRevalidateString = "must-revalidate";
-        private const string NoCacheString = "no-cache";
-        private const string NoStoreString = "no-store";
-        private const string NoTransformString = "no-transform";
-        private const string OnlyIfCachedString = "only-if-cached";
-        private const string PrivateString = "private";
-        private const string ProxyRevalidateString = "proxy-revalidate";
-        private const string PublicString = "public";
-        private const string SharedMaxAgeString = "s-maxage";
```

Comment:
Also avoiding a ToLower() which is required since switch on strings are case-sensitive. I'm seeing a one percent performance increase in ResponseCaching with it. Also it's code within a private method, so I'm not too concerned about its readability.